### PR TITLE
updating RegisterView.tsx

### PR DESCRIPTION
**Pull request by shubhay09**
hovering function added as well as red highlight to let users now its important 

Signed-off-by: shubhay09 <82525849+shubhay09@users.noreply.github.com>

### CODE REVIEW

**Comment by elliotcourant** [comment]
I'd remove the red highlight please, as the form item is still required for submission.

Could you change the underline to be `focus:underline` and add a `focus:text-blue-500` to each link as well?

File: ui/components/Authentication/Registration/RegisterView.tsx [hunk 1]
```diff
@@ -296,18 +296,22 @@ export default function RegisterView(): JSX.Element {
                         }
                         label={
                           <Fragment>
+                            <a
+                            className="hover:text-red-500">
                             I agree to monetr's&nbsp;
                             <a
-                              className="hover:text-blue-500 hover:underline"
+                              className="hover:text-blue-500 underline hover:underline text-black-500"
```
Comment (elliotcourant) [inline ui/components/Authentication/Registration/RegisterView.tsx:303]: https://tailwindcss.com/docs/text-color Also there is not a `text-black-500` class for tailwind?

**Comment by shubhay09** [comment]
i have made the changes you can check

**Review by shubhay09 — COMMENTED** [review]
edited correctly

**Review by shubhay09 — COMMENTED** [review]
checked the changes

**Pull request closed without merging**